COPY/etc/httpd/conf.d/manageiq-redirects-ui

```diff
@@ -9,3 +9,5 @@ RewriteCond %{REQUEST_URI} !^/ansibleapi
 RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-f
 RewriteRule ^/ balancer://evmcluster_ui%{REQUEST_URI} [P,QSA,L]
 ProxyPassReverse / balancer://evmcluster_ui/
+ProxyPass /ws/notifications balancer://evmcluster_ui_ws/ws/notifications
```

Comment:
before, `/ws` all went to the console "websocket worker"
after: `/ws/notifications` are going to the ui worker.
can't say ui worker (because that is http specific), so we had to invent a name ui_ws worker, which is the ui worker, just speaking ws